easybuild/easyconfigs/g/GPAW/GPAW-19.8.1-intel-2018b-ASE-3.18.0-Python-3.6.6.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = "PythonPackage"
+
+name = 'GPAW'
+version = '19.8.1'
+_aseversion = '3.18.0'
+_pythonsuffix = '-Python-%(pyver)s'
+versionsuffix = '-ASE-' + _aseversion + _pythonsuffix
+
+homepage = 'https://wiki.fysik.dtu.dk/gpaw/'
+description = """GPAW is a density-functional theory (DFT) Python code based on the projector-augmented wave (PAW)
+ method and the atomic simulation environment (ASE). It uses real-space uniform grids and multigrid methods or
+ atom-centered basis-functions."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+patches = [
+    ('GPAW-19.8.1-customize-intel.patch', 0),
+]
+checksums = [
+    '79dee367d695d68409c4d69edcbad5c8679137d6715da403f6c2500cb2178c2a',  # gpaw-19.8.1.tar.gz
+    'fd3668ae52683d4e36e9c8b4e3e69faa93d949601d1128662dc31be61876610c',  # GPAW-19.8.1-customize-intel.patch
+]
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('ASE', _aseversion, _pythonsuffix),
+    ('libxc', '3.0.1'),
```

Comment:
@schiotz Existing easyconfigs for `GPAW` 19.8.1 were using `libxc` 4.3.4.

Is there any reason you're not using `libxc` 4.3.4 here, but 3.0.1 instead?

Reply:
Note that the existing easyconfigs for `libxc` 4.2.3 that use a `2018b` toolchain would do?

Reply:
@boegel 
The only reason for using libxc 3.0.1 is that some other software uses that version, so a conflict occurs if I switch to 4.2.3.  Other than that, 4.2.3 would be preferable.

Reply:
You mean `AtomPAW` and `ABINIT`?

Maybe it's good to mention that in a comment above this line?

Reply:
Yes, that's the ones.  I will add a comment.  In the 2019a toolchains we use a newer libxc.